backend/src/models/Article.ts

```diff
@@ -0,0 +1,30 @@
+import { InferSchemaType, Schema, model } from "mongoose";
+
+const articleSchema = new Schema(
+  {
+    _id: {
+      type: Schema.Types.ObjectId,
+      required: true,
+    },
```

Comment:
MongoDB will automatically generate _id so I think its cleaner to just remove this part.